src/ripple/nodestore/impl/DatabaseRotatingImp.cpp

```diff
@@ -143,10 +143,6 @@ DatabaseRotatingImp::fetchNodeObject(
         switch (status)
         {
             case ok:
-                ++fetchHitCount_;
-                if (nodeObject)
-                    fetchSz_ += nodeObject->getData().size();
-                break;
```

Comment:
We lost additions to both `fetchHitCount_` and `fetchSz_` here. I'm fine with that as long as that's really what you want and not an unintended side effect.

Also, I'd prefer for you to include a `break;` after `case ok:`, rather than falling through.  The generated code will be identical.  Falling through a `case` should be unusual and require a comment.  Putting in the `break;` leaves us in a better position for future maintenance.